recipes/squash/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ex
+
+EXTRA_CMAKE_ARGS=""
+if [[ `uname` == 'Darwin' ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+  EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}"
+  EXTRA_CMAKE_ARGS="${EXTRA_CMAKE_ARGS} -DCMAKE_MACOSX_RPATH=ON"
+else
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+# This flags come from project's travis build.
+export CFLAGS="${CFLAGS} -fno-omit-frame-pointer -fstack-protector-all"
+export CXXFLAGS="${CXXFLAGS} -fno-omit-frame-pointer -fstack-protector-all"
+eval export $LIBRARY_SEARCH_VAR=${PREFIX}/lib
```

Comment:
This is also getting phased out. Setting the rpath explicitly (e.g. `-Wl,-rpath,${PREFIX}/lib`) is the recommended way to go and seems to work fine. The main driver behind this is the addition of SIP on 10.11. While we are not building on 10.11 yet, it is a matter of time before we do.

xref: https://github.com/conda-forge/conda-forge.github.io/issues/238